Muehle_Logic.ipynb

```diff
@@ -198,66 +198,65 @@
    "outputs": [],
    "source": [
     "def make_move(ring, cell):\n",
-    "    global Status\n",
-    "    phase = get_player_phase(Status.state, Status.current_player)\n",
-    "    possible_positions = next_positions(Status.state, Status.current_player, ring, cell)\n",
+    "    global game\n",
```

Comment:
Müsste game nicht eigentlich groß geschrieben werden? Hatte wir ja schon bei status->Status.

Reply:
Gegen die Änderung von Staus auf Game hab ich nichts.

Reply:
Da war ich mir auch sehr unsicher...
Aber eigentlich ist game ja dann eine Instanz/Variable... Ich les nochmal nach.

Ich habe es übrigens umbenannt, weil Status ja zum einen nicht Englisch ist und zum anderen wollte ich keine Verwirrung mit state

Reply:
Stimmt. Mit State wäre das dann schwierig.

Reply:
[Hier](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) steht für Instanzen einer Klasse folgendes:
![image](https://user-images.githubusercontent.com/54306427/109481003-86246e80-7a7c-11eb-80ad-51b2bb3f074b.png)

Also ist `game = Game()` korrekt.